doc/src/fix_brownian.rst

```diff
@@ -0,0 +1,151 @@
+.. index:: fix brownian
+
+fix brownian command
+====================
+
+Syntax
+""""""
+
+.. parsed-literal::
+
+   fix ID group-ID brownian gamma_t diff_t seed keyword args
+
+* ID, group-ID are documented in :doc:`fix <fix>` command
+* brownian/sphere = style name of this fix command
+* gamma_t = translational friction coefficient
+* diff_t = translational diffusion coefficient
+* zero or more keyword/value pairs may be appended
+* keyword = *rng* 
+
+  .. parsed-literal::
+
+        *rng* value = *uniform* or *gaussian* or *none*
+         *uniform* = use uniform random number generator
+         *gaussian* = use gaussian random number generator
+         *none* = turn off noise
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix 1 all brownian 1.0 3.0 1294019
+   fix 1 all brownian 1.0 3.0 19581092 rng none 
+   fix 1 all brownian 1.0 3.0 19581092 rng uniform
+   fix 1 all brownian 1.0 3.0 19581092 rng gaussian
+
+
+Description
+"""""""""""
+
+Perform Brownian Dynamics integration to update position and velocity
+of atoms in the group each timestep. Brownian Dynamics uses Newton's laws of
+motion in the limit that inertial forces are negligible compared to
+viscous forces. The stochastic equation of motion is
+
+.. math::
+
+   dr = \frac{F}{\gamma_t}dt+\sqrt{2D_t}dW_t, \\
+
+where :math:`dW_t` is a Wiener processes (see e.g. :ref:`(Gardiner) <GardinerC1>`).
+
+.. note::
+   This integrator is designed for generic non-equilibrium
+   simulations with additive noise. There are two important cases which
+   (conceptually) reduce the number of free parameters in this fix.
+   (a) In equilibrium simulations
+   (where fluctuation dissipation theorems are obeyed), one can define
+   the thermal energy :math:`k_bT=D_t\gamma_t`.
+   
+---------
+
+.. note::
+   Temperature computation using the :doc:`compute temp <compute_temp>`
+   will not correctly compute temperature of these overdamped dynamics
+   since we are explicitly neglecting inertial effects.
+   See e.g. chapter 6 of :ref:`(Doi) <Doi1>` for more details on this.
+   Temperature is instead defined in terms of the note above (for
+   equilibrium systems).
+
+---------
+
+.. note::
+   The diffusion coefficient :math:`D_t` is measured
```

Comment:
the units of keywords are more commonly given where the keywords are listed and not in a "Note" section.

Reply:
Done.